File: ADS 2023.2/AlgoritimosProgramacao/Algoritmos/While/lista_fabio/AlgoritimosProgramacao/Algoritmos/Questões/List-01/q37.py
def main():
    #Input

    total_days = int(input("Enter the age in days: "))

    #Processing

    years = round(total_days / 365)
    months = round((total_days % 365) / 30)
    days = (total_days % 365) % 30

    #Output    
    print(f"The value of the days entered converted in years, months and days is equal to: {years} anos, {months} mes and {days} dias.")
main()

File: ADS 2023.2/AlgoritimosProgramacao/Algoritmos/While/lista_fabio/AlgoritimosProgramacao/Algoritmos/Questões/List-01/q37.js
import { question } from "readline-sync";
function main(){
    //Input
    const total_days = parseInt(question("Enter the age in days: "));

    //Processing

    const years = Math.floor(total_days / 365)
    const months = Math.floor((total_days % 365)/30)
    const days = ((total_days % 365) % 30)

    //Output    
    console.log(`The entered value of the days converted in years, months and days is equal: ${years} years, ${months} mes and ${days} days`)

}
main()